### PR TITLE
Hauki 81 page structure

### PR DESCRIPTION
**Pull request by VilleMiekkoja**
Jatkojirattu footer: https://helsinkisolutionoffice.atlassian.net/browse/HAUKI-97

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -1,38 +1,7 @@
...
-  }
-  to {
-    transform: rotate(360deg);
-  }
-}
+  height: 100%;
+  margin: 0;
+  display:flex;
+  flex-direction:column;
+}
```
Comment (minevala) [inline src/App.css:7]: These issues can be avoided by adding insert_final_newline into editorConfig https://github.com/editorconfig/editorconfig/wiki/EditorConfig-Properties#insert_final_newline

File: src/components/footer/HaukiFooter.css [hunk 2]
```diff
@@ -0,0 +1,7 @@
+.page-footer {
+  margin-top: auto;
+}
+
+.hauki-koros pattern {
+  fill: #6feed3;
```
Comment (minevala) [inline src/components/footer/HaukiFooter.css:6]: Should we have some specific place for these theme-specific values?

File: src/components/navigation/HaukiNavigation.tsx [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+      )}
+
+      <Navigation.Actions>
+        <Navigation.User
+          authenticated={authenticated}
+          label="Kirjaudu"
+          onSignIn={(): void => setAuthenticated(true)}
+          userName="John Doe">
+          <Navigation.Item
+            label="Your profile"
```
Comment (minevala) [inline src/components/navigation/HaukiNavigation.tsx:47]: Should we have all these label texts in Finnish at first?

File: src/components/footer/HaukiFooter.css (hunk 2 above)
Comment (minevala) [reply src/components/footer/HaukiFooter.css:6]: Maybe some theme.css or variables.css?

File: src/components/navigation/HaukiNavigation.tsx (hunk 3 above)
Comment (VilleMiekkoja) [reply src/components/navigation/HaukiNavigation.tsx:47]: Good idea, I'll change those.

- [ ] Change label texts to Finnish

File: src/components/footer/HaukiFooter.css (hunk 2 above)
Comment (VilleMiekkoja) [reply src/components/footer/HaukiFooter.css:6]: Yes good idea!

- [ ] Separate file for theme-specific css variables

File: src/App.css (hunk 1 above)
Comment (VilleMiekkoja) [reply src/App.css]: - [ ] Insert missing empty lines

**Review by minevala — APPROVED** [review]
<details>
<summary>
<i>Good job 👍  </i>
</summary>
🏅 
</details>

**Pull request merged**